docs/body.md

```diff
@@ -748,39 +615,336 @@ when this happens.
+            PipelinePublishers:
+              - https://10.0.0.10:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
+              - https://10.0.0.12:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
+            ManageEventPipeline: true
+            EventPipelinePublishers:
+              - https://10.0.0.10:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
+              - https://10.0.0.12:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
+
+*   This template must be ***rendered*** into your templates before you initiate a deployment from
+    Undercloud.<br>
+    For more information on RHOSP template management, see RedHat's
+    [Including Environment Files in Overcloud Creation](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/advanced_overcloud_customization/index#sect-Including_Environment_Files_in_Overcloud_Creation).
+
+*   After deployment is complete, go to the undercloud, SSH into controller and go
+    to `/etc/ceilometer` in ceilometer_agent_notification container to check if
+    `pipeline.yaml` and `event_pipeline.yaml` file is updated:
+
+            ssh heat-admin@<controller ip>
+            sudo su -
+            sudo docker exec --user 0 -it ceilometer_agent_notification /bin/bash
+            # Ensure your configuration had the right publishers
+
+#### Notes:
+
+*   Events exposed by Ceilometer are listed in zOpenStackProcessEventTypes.
+    By default, this is set to the single *compute.instance.create.error* .
+
+*   Proxy warnings in [ZPS-4689](#known-issues) may be displayed during the
+    upgrade to 3.0.0 if you have multiple collectors. These can be safely
+    ignored.
+
+*   The Instance metrics for *Disk IO Rate* are deprecated in OpenStack version
+    Queens and later. Collection for those metrics will be missing.
+    Future OpenStack releases will remove these metrics and graphs completely.
+
+    In the meantime, if you still require these metrics, you can edit the
+    OpenStack Ceilometer configuration /etc/ceilometer/polling.yaml and add the
+    following to the *meters* section:
+
+            - disk.read.bytes
+            - disk.read.requests
+            - disk.write.bytes
+            - disk.write.requests
+
+    - Ensure the resulting YAML syntax is valid restart the ceilometer-polling service
+
+    Finally note that these metrics ARE deprecated and will be removed in
+    future releases of OpenStack itself.
+
+    Note: **There is no known template configuration for this for RHOSP
+          templates at this time and are unsupported.**
+
+*   Make sure you restart ceilometer-notification service whenever you change
+    `pipeline.yaml` or `event_pipeline.yaml`.
+
+*   For Queens and Rocky devices, make sure to restart ceilometer-polling service
+    if you have updated `/etc/ceilometer/polling.yaml`
+
+*   The RabbitMQ-Ceilometer service
+
+    `RabbitMQ-Ceilometer` is typically only needed for legacy (pre 3.0,0 ZenPack) systems.
```

Comment:
We need to work on this section.    This service is "needed" in 3.0.0 as well (unfortunately) in that you have to have it running, even if it isn't used.   It may be better to remove this section anyway and write something that just explains in broad terms how we integrate with ceilometer.   (unless we already have that explained somewhere)

Reply:
JOsh, for now, I will remove that false statement and release. Lets make further changes later after release.